src/area-brush.c

```diff
@@ -198,7 +175,7 @@ LIBUI_FUNCTION(getStops) {
 
 	napi_value constructor, array;
 
-	napi_status status = napi_get_reference_value(env, AreaDrawBrushGradient, &constructor);
+	napi_status status = napi_get_reference_value(env, BrushGradientStop, &constructor);
 	CHECK_STATUS_THROW(status, napi_get_reference_value);
 
 	status = napi_create_array_with_length(env, brush->NumStops, &array);
```

Comment:
It seems that setStops accept [external] objects representing uiDrawBrushGradientStop,
while getStops returns instances of BrushGradientStop

Reply:
Yes. Handled here:

```js
	/**
	 * Set the gradient stops
	 * @param {Array<BrushGradientStop>} value - the gradients stops
	 * @return {string}
	 */
	set stops(value) {
		AreaBrush.setStops(this.handle, value.map(x => x.handle));
	}

	/**
	 * Get the gradient stops
	 * @return {Array<BrushGradientStop>}
	 */
	get stops() {
		return AreaBrush.getStops(this.handle);
	}
```